docassemble/assemblylinewizard/data/questions/pdf_field_tester.yml

```diff
@@ -57,11 +50,11 @@ subquestion: |
 
   Field name (bold if {reserved}) | Field type | {Internal DA name} | Max length
   -----------|------------|-------------------|-----------
-  % for field in fields:
+  % for field in fields_raw:
   ${":exclamation-triangle: " if ' ' in field[0] else ''} ${":exclamation-circle: " if len(remove_multiple_appearance_indicator(varname(field[0]))) == 0 else ''} ${'**' + str(field[0]) + '**' if is_reserved_label(field[0]) else field[0]} | ${ pdf_field_type_str(field) } (${field[4]}) | ${map_raw_to_final_display(field[0]) if not (map_raw_to_final_display(field[0]) == field[0]) else ''} | ${ get_character_limit(field) if get_character_limit(field) else 'n/a' }
   % endfor
 
-  The following are signature fields: ${comma_and_list(signature_fields)} 
+  The following are signature fields: ${comma_and_list(signature_fields_test)} 
   
   The following are checkbox fields: ${comma_and_list(checkbox_fields)}
```

Comment:
I know you didn't change this, but these two "The following are ..." statements should be wrapped in a
```
% if len(signature_fields_test) > 0:
...
% end if
```

Reply:
Again, I'll leave it to #391 as it could change the code significantly.